packages/lit-dev-content/rollup.config.js

```diff
@@ -127,4 +129,33 @@ export default [
       }),
     ],
   },
+
+  /**
+   * Creates an iife for scripts that need to be inlined with inlinesyncjs and
```

Comment:
it's the only output mode I can find on rollup that will bundle everything into a single file. In case somebody down the line wants to use the `inlinesyncjs` macro they don't have to worry about imports

Reply:
actually it's not necessary anymore removing